library/core/class.smarty.php

```diff
@@ -34,7 +34,7 @@ public function init($Path, $Controller) {
 
         // Get an ID for the body.
         $BodyIdentifier = strtolower($Controller->ApplicationFolder.'_'.$ControllerName.'_'.Gdn_Format::alphaNumeric(strtolower($Controller->RequestMethod)));
-        $Smarty->assign('BodyID', $BodyIdentifier);
+        $Smarty->assign('BodyID', Gdn_Format::text($BodyIdentifier));
```

Comment:
I don't think so; I only format in the PHP-based admin.master.php which doesn't go thru smarty.